templates/post-install-job.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.defaultCollector.enabled }}
+apiVersion: batch/v1
```

Comment:
So it doesn't work to do it not as a post-install-job because of the need for the CRD to be installed first?

Reply:
it's created as a post-install job due to needing the MDAI operator (and associated webhooks) up & running. it _could_ be added to the operator helm chart instead of the overall

Reply:
Ah yea, probably best in the operator? But would require a separate PR and release, so this is good.